pkg/shell/shell.go

```diff
@@ -116,7 +116,7 @@ func (sp ShellProxy) ConnectFunc(w http.ResponseWriter, r *http.Request) {
 
 	// now use the secret and ssh off to something
 	config := &ssh.ClientConfig{
-		User: "ubuntu",
+		User: vm.Spec.SshUsername,
```

Comment:
Maybe the default logic should also be put here, that way no VM is broken accidentally